File: CS/MyMvpvmApplication/Views/TaskCollectionView.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using MyMvpvmApplication.ViewModels;
using DevExpress.Mvvm.POCO;

namespace MyMvpvmApplication.Views {
    public partial class TaskCollectionView : UserControl {
        public TaskCollectionView() {
            InitializeComponent();
            viewModelCore = ViewModelSource.Create<TaskCollectionViewModel>();
        }
        TaskCollectionViewModel viewModelCore;
        public TaskCollectionViewModel ViewModel {
            get { return viewModelCore; }
        }

        protected override void OnLoad(System.EventArgs e) {
            base.OnLoad(e);
            taskBindingSource.DataSource = ViewModel.Tasks;
        }
    }
}

File: VB/Views/TaskCollectionView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports MyMvpvmApplication.ViewModels
Imports DevExpress.Mvvm.POCO

Namespace MyMvpvmApplication.Views
	Public Partial Class TaskCollectionView
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
			viewModelCore = ViewModelSource.Create(Of TaskCollectionViewModel)()
		End Sub
		Private viewModelCore As TaskCollectionViewModel
		Public ReadOnly Property ViewModel() As TaskCollectionViewModel
			Get
				Return viewModelCore
			End Get
		End Property

		Protected Overrides Sub OnLoad(ByVal e As System.EventArgs)
			MyBase.OnLoad(e)
			taskBindingSource.DataSource = ViewModel.Tasks
		End Sub
	End Class
End Namespace
